src/layouts/EmptyLayout.tsx

```diff
@@ -1,11 +1,19 @@
 import styled from "styled-components"
+import { Header } from "../components/Header"
+import * as React from 'react';
 
 // Layout for public pages
-interface layoutProps{
-  children : any
+interface LayoutProps {
+  children: React.ReactNode
```

Comment:
```suggestion
  children: ReactNode
```
dùng hàm nào thì import hàm đấy thôi, khỏi phải import nguyên cả React